CamTool/Assets/Scripts/cameraPicture.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class cameraPicture : MonoBehaviour
```

Comment:
Camel case the name of this class. Both file and class name should be "CameraPicture"